src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1488,11 +1488,13 @@ static void _copyAttr(SvgNode* to, SvgNode* from)
         case SvgNodeType::Polygon: {
             to->node.polygon.pointsCount = from->node.polygon.pointsCount;
             to->node.polygon.points = (float*)calloc(to->node.polygon.pointsCount, sizeof(float));
```

Comment:
This case, memory reset is not necessary, please replace calloc() to malloc()

Reply:
I add a patch to fix this.